### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sophie-Xie**
update for move repo

### CODE REVIEW

**Review by yixinglu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ Compared with other graph database solutions, **Nebula Graph** has the following
```
Comment (jamieliu1023) [inline README.md:33]: I prefer that we use clear instructional words to tell people what to do and where to go:
1. This repo hosts the code of versions prior to v1.2.1 and v2.5.0 later
2. If you are looking to use the versions between v2.0.0 and v2.5.0, please head to [nebula-graph](https://github.com/vesoft-inc/nebula-graph)   
3. The data format, rpc protocols, clients, etc. are not compatible between Nebula Graph v1.x and v2.x

Comment (Sophie-Xie) [reply README.md:33]: Thanks, I modify the readme.

File: README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Compared with other graph database solutions, **Nebula Graph** has the following
 
 The Nebula Graph team is glad to announce the Nebula Graph 2.0.0 GA. This is a brand new start with several important updates, and more of them are on the way. Welcome to join us.
 
-This repo hosts the code of versions prior to v1.2.1 and v2.5.0 later. If you are looking to use the versions between v2.0.0 and v2.5.0, please head to [Nebula Graph repo](https://github.com/vesoft-inc/nebula-graph). 
+This repo hosts the code of versions prior to v1.x and v2.5.0 later. If you are looking to use the versions between v2.0.0 and v2.5.0, please head to [Nebula Graph repo](https://github.com/vesoft-inc/nebula-graph).
```
Comment (randomJoe211) [inline README.md:36]: "prior to" indicates before. Readers might mistake it for the 0.x versions. Use "prior to v2.0.0 alpha" or "This repository hosts the source code of Nebula Graph versions before 2.0.0-alpha and after v2.5.0."

Comment (Sophie-Xie) [reply README.md]: Thanks, accepted and modified it.

**Review by randomJoe211 — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**